compose/project.py

```diff
@@ -278,10 +278,10 @@ def restart(self, service_names=None, **options):
         for service in self.get_services(service_names):
             service.restart(**options)
 
-    def build(self, service_names=None, no_cache=False, pull=False):
+    def build(self, service_names=None, force_rm=False, no_cache=False, pull=False):
         for service in self.get_services(service_names):
             if service.can_be_built():
-                service.build(no_cache, pull)
+                service.build(force_rm, no_cache, pull)
```

Comment:
Minor note here: If someone is using compose as a library (which we don't officially support), and they're incorrectly using positional args for kwargs (like we do here), adding the new kwarg to the beginning of the arg list will break code calling this function.

Adding it to the end of the arg list is a bit nicer.